q2cli/tools.py

```diff
@@ -145,6 +143,62 @@ def peek(path):
         click.secho(metadata.format)
 
 
+@tools.command('inspect-metadata',
+               short_help='Inpect columns available in metadata.')
+@q2cli.option('--tsv/--no-tsv', default=False)
+@click.argument('path', type=click.Path(exists=True, file_okay=True,
+                                        dir_okay=False, readable=True))
+@q2cli.util.pretty_failure(traceback=None)
+def inspect_metadata(path, tsv, failure):
+    import qiime2
+    import qiime2.sdk
+
+    try:
+        artifact = qiime2.sdk.Result.load(path)
+    except Exception:
+        metadata = qiime2.Metadata.load(path)
+    else:
+        if isinstance(artifact, qiime2.Visualization):
+            raise Exception("Visualizations do not have metadata.")
```

Comment:
Can you update this text to be consistent with the error message used in the MetadataHandler / MetadataColumnHandler when something can't be viewed as metadata? That'll also match the language in the `docs` and framework's error messages.

"Visualizations cannot be viewed as QIIME 2 Metadata."